src/sharedHelpers/breakpoint.ts

```diff
@@ -0,0 +1,29 @@
+import screens from "styles/tailwindScreens";
+
+export const BREAKPOINTS
+  = Object.fromEntries(
+    Object.entries( screens || {} ).map( ( [breakpoint, widthString] ) => {
+      if ( typeof widthString !== "string" ) {
+        throw new Error( `Unexpected breakpoint value: ${widthString}` );
+      }
```

Comment:
<img width="586" height="83" alt="Image" src="https://github.com/user-attachments/assets/a0c448f3-a9e4-456b-a746-609c32555a5a" />

The typing of `screens` is somewhat complicated because Tailwind allows so many different formats for the width value of the breakpoint config. And because `parseInt` requires a `string` be passed in, we need to ensure that `widthString` is indeed a `string`, hence this line.